hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/XceiverClientGrpc.java

```diff
@@ -306,7 +306,7 @@ private XceiverClientReply sendCommandWithRetry(
     } else {
       Preconditions.checkNotNull(ioException);
       LOG.error("Failed to execute command " + request + " on the pipeline "
```

Comment:
Also if we can change this to parameterized logging.

Reply:
Addressed in the latest patch.